src/helics/network/networkDefaults.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2017-2021,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable
+Energy, LLC.  See the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "networkDefaults.hpp"
+
+#include "../helics_enums.h"
+
+namespace helics {
+int getDefaultPort(int coreType)
+{
+    switch (coreType) {
+        case HELICS_CORE_TYPE_TCP:
+            return network::DEFAULT_TCP_BROKER_PORT_NUMBER;
+        case HELICS_CORE_TYPE_TCP_SS:
+            return network::DEFAULT_TCPSS_PORT;
```

Comment:
Seems a bit strange using different naming conventions here -- `DEFAULT_<coretype>_PORT` instead of `DEFAULT_<coretype>_BROKER_PORT_NUMBER`. I kind of prefer the naming used for the TCPSS core, maybe aliases with the same naming could be added for the other core types in the networking namespce.

Reply:
renamed them